webapp/sources/rudder/rudder-core/src/main/resources/reportsSchema.sql

```diff
@@ -121,12 +121,14 @@ CREATE TABLE ReportsExecution (
 , complete     boolean NOT NULL
```

Comment:
we discussed that, and a run could have a bug and be incomplete. But that notion of incomplete is totally different from the old notion of "we didn't get all reports yet". So to avoid confusion (and bug mixing one notion with the other), we will remove that old `complete` notion, and we will treat broken runs with a dedicated state of `RunAndConfigInfo`.